torch/csrc/cuda/Module.cpp

```diff
@@ -409,7 +417,16 @@ PyObject * THCPModule_initExtension(PyObject *self)
   if (!THCPModule_initCuda(torch_module)) {
     return NULL;
   }
-  Py_RETURN_NONE;
+
+  auto m = py::handle(torch_module).cast<py::module>();
+  py::class_<cudaDeviceProp>(m,"CudaDeviceProperties")
```

Comment:
Can you rename this to `_CudaDeviceProperties`? We don't want people to ever think it is a public thing they can construct